requirements-ci.txt

```diff
@@ -1,10 +1,10 @@
-numpy
-scipy<1.13.0
+numpy~=2.0.0
+scipy<=1.13.0
 cvxpy
 cvxopt
 networkx
 rustworkx
-autograd
+autograd~=1.7.0
```

Comment:
No issues when running the latest autograd against 1.x python?

Reply:
Against Numpy 1.x? I don't think we have a separate workflow to test the newest autograd with the older number versions.